ocaml/idl/datamodel.ml

```diff
@@ -6577,6 +6577,17 @@ let vdi_disable_cbt = call
     ~allowed_roles:_R_VM_ADMIN
     ()
 
+let vdi_set_cbt_enabled = call
+    ~name:"set_cbt_enabled"
+    ~in_oss_since:None
+    ~in_product_since:rel_inverness
+    ~params:[Ref _vdi, "self", "The VDI for which CBT enabled status should be set";
+             Bool, "value", "The value to set"]
+    ~errs:[]
+    ~hide_from_docs:true
+    ~allowed_roles:_R_POOL_ADMIN
```

Comment:
Does it work if we change this to `_R_LOCAL_ROOT_ONLY`? I think that's what we should use if SM uses the local unix domain socket and if that has root privileges.

Reply:
well I accidentally changed the allowed roles for get_nbd_info instead but I was able to run the sommand from dom0 so this should work

Reply:
I think we login as local root from SM:
https://github.com/xapi-project/sm/blob/master/drivers/SR.py#L97